account.go

```diff
@@ -111,6 +112,33 @@ type Account struct {
 	Deleted        bool     `json:"deleted"`
 }
 
+type ExternalAccounts struct {
```

Comment:
I think in this particular case it might be a good idea to re-use the fields in `ListMeta` and to rename `Data` to `Values` for consistency. See [this example in cards](https://github.com/stripe/stripe-go/blob/5d35501411915df467df98dcfebabfde5bba9813/card.go#L68,L71).